qvisor/src/vmspace/xpu/cuda_mem_manager.rs

```diff
@@ -0,0 +1,276 @@
+use crate::vmspace::{CUDA_MEMCPY_DEVICE_TO_HOST, CUDA_MEMCPY_HOST_TO_DEVICE};
+use crate::xpu::cuda_api::*;
+use std::collections::BTreeMap;
+use std::sync::atomic::{AtomicU32, Ordering};
+use cuda_runtime_sys::cudaStream_t;
+
+pub const ONE_TB: u64 = 1 << 40; //0x10_000_000_000;
```

Comment:
Duplication of [declaration](https://github.com/QuarkContainer/Quark/blob/922cfe712075a0ce41024cfe1a566724fb2da79f/qlib/linux_def.rs#L2953).